Examples/RKSearchExample/RKSearchExample.xcodeproj/project.pbxproj

```diff
@@ -338,7 +338,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
```

Comment:
We also have a requirement of iOS8.0 and above. Although we'd love to only have to support 8.4.1, we have outside factors that are keeping us on 8.0. Is it possible that we can lower the minimum deployment target to 8.0?